scripts/setup-env.sh

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env bash
-CHECK="test \"$_\" != \"$0\""
-$CHECK || { echo >&2 "Use this script by sourcing it \`. $0\` instead"; exit 1; }
+[[ "$_" != "$0" ]] || { echo >&2 "Use this script by sourcing it \`. $0\` instead"; exit 1; }
```

Comment:
This doesn't work though.

Reply:
There seem to be [better ways](https://stackoverflow.com/questions/2683279/how-to-detect-if-a-script-is-being-sourced#34642589) of doing it anyway, should we switch?

Reply:
It does, sorry

Reply:
It doesn't ran it with `sh scripts/setup-env.sh` and that didn't work before either

Reply:
Cool hack using `return`, yeah much more robust.